_pages/ingest/mlcp.md

```diff
@@ -27,6 +27,13 @@ The `-transform_module` and `-transform_namespace` parameters must be set to the
 -transform_namespace "http://marklogic.com/data-hub/mlcp-flow-transform"
 </pre>
 
+For SJS transforms use
+
+<pre class="cmdline">
+-transform_module "/com.marklogic.hub/mlcp-flow-transform.sjs"
```

Comment:
I don't think so. :) Shouldn't that be: /MarkLogic/data-hub-framework/transforms/mlcp-flow-transform.sjs?

Reply:
This is still wrong, isn't it?